src/common/snippets/tests/include/lowered/pass/optimize_domain.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <common_test_utils/ov_test_utils.hpp>
+#include "snippets/shape_types.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+struct OptimizeDomainParams {
+    OptimizeDomainParams() = default;
+    OptimizeDomainParams(size_t, size_t, std::vector<ov::PartialShape>, ov::snippets::VectorDims, size_t);
+    size_t min_jit_work_amount = 0;                   // Min jit work amount
+    size_t min_parallel_work_amount = 0;              // Min parallel work amount
+    std::vector<ov::PartialShape> input_shapes;       // Input shapes
+    ov::snippets::VectorDims exp_master_shape;        // Expected master_shape
+    size_t exp_loop_depth = 0;                        // Expected loop depth (aka tile rank)
```

Comment:
The purpose of the structure's fields is already obvious from their name. Do we really need explanatory comments?

Reply:
I agree that they are a little bit excessive, but I wanted to correspond with other tests. Anyway, let's remove all except for the last two. After all, how can one know what  `exp_master_shape` means? Is it experimental master shape or may be exponential? Or even expanded 😁

Reply:
Good point) Ok, let's leave 2 last comments